docs/bookstore/page-1/send-review-comment-to-broker.md

```diff
@@ -0,0 +1,423 @@
+    'ce-specversion': receivedEvent.specversion,
+    'ce-type': receivedEvent.type,
+    'ce-source': receivedEvent.source,
+    'ce-id': receivedEvent.id,
+  },
+  body: JSON.stringify(receivedEvent.data),
+});
+```
+
+![Image](images/image21.png)
+
+**Exploring Other Functions**
+
+The Node.js server contains other functions that follow a similar pattern, with detailed comments explaining their functionality:
+
+- `/insert`: Receives CloudEvents and inserts the payload into the PostgreSQL database.
+- `/comment`: Creates a WebSocket connection with the frontend to transport comments from the database.
+
+### **Step 2: Create Broker**
+
+![Image](images/image17.png)
+
+The Broker acts as a router in your event-driven application, receiving events and routing them to the correct destination.
+
+
+
+- 1: Create a new YAML file named `node-server/config/200-broker.yaml` and add the following content:
+
+???+ abstract "_node-server/config/200-broker.yaml_"
+    ```yaml
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+        name: bookstore-broker
+    ```
+
+- 2: Apply the YAML file:
+
+```bash
+kubectl apply -f node-server/config/200-broker.yaml
+```
+
+You will see the following output:
+```
+broker.eventing.knative.dev/bookstore-broker created
+```
+
+Alternatively, use the [Knative CLI `kn`](https://knative.dev/docs/client/#kn){:target="_blank"} to create the Broker:
+
+```bash
+kn broker create bookstore-broker
+```
+
+You will see the following output:
+```
+Broker 'bookstore-broker' successfully created in namespace 'default'.
+```
+
+???+ success "Verify"
+    Running the following command to list the Brokers:
+    ```bash
+    kubectl get brokers
+    ```
+
+    
+    You should see the Broker `bookstore-broker` with `READY` status as `True`.
+
+    ```
+    NAME               URL                                                                                 AGE     READY   REASON
+    bookstore-broker   http://broker-ingress.knative-eventing.svc.cluster.local/default/bookstore-broker   7m30s   True    
+    ```
+
+
+
+
+???+ bug "Troubleshooting"
+
+    If there are issues, use the following command to diagnose:
+
+    ```bash
+    kubectl describe broker bookstore-broker
+    ```
+
+### **Step 3: Create a SinkBinding between the Node.js server and Broker**
+
+![Image](images/image15.png)
+
+Hardcoding URLs to connect with Kubernetes services in your application can be limiting and inflexible. A SinkBinding dynamically injects the URL of the Kubernetes service into your application.
+
+Learn more about SinkBinding [here](https://knative.dev/docs/eventing/custom-event-source/sinkbinding/){:target="_blank"} and the [spec schema](https://knative.dev/docs/eventing/custom-event-source/sinkbinding/reference/){:target="_blank"}!
+
+**Create a SinkBinding:**
+
+- 1: Create a new YAML file named `300-sinkbinding.yaml` in the `node-server/config` folder and add the following content:
```

Comment:
please use the full path (`node-server/config/300-sinkbinding.yaml`), yes that is repaetive but much nice for copy paste.